src/__test__/redux/actions/cellInfo/updateCellInfo.test.js

```diff
@@ -10,7 +10,7 @@ const mockStore = configureStore([thunk]);
 
 describe('updateCellInfo action', () => {
   const updatePayload = {
-    cellName: 'C1',
+    cellId: 'C1',
```

Comment:
While you're making this change, can you also fix the relative import paths to use absolute import paths in this file?